templates/code-style-validation.yml

```diff
@@ -28,6 +32,8 @@ jobs:
     displayName: Validate code
 
   - bash:
-      pip install git+https://github.com/shotgunsoftware/tk-toolchain.git#egg=tk-toolchain &&
-      ((test -e docs && tk-docs-preview --build-only) || echo No "docs" folder detected.)
+      # PySide 2 and tk-toolchain are required to build the documentation.
+      pip install PySide2 git+https://github.com/shotgunsoftware/tk-toolchain.git@${{ parameters.tk_toolchain_ref }}#egg=tk-toolchain &&
+      git clone https://github.com/shotgunsoftware/tk-core --depth 1 --branch SG-13264-python-3-compat ../tk-core &&
```

Comment:
Does this warrant a TODO or a comment about why we're using a branch of tk-core for now?

Reply:
It should, but I already have another branch ready for review that addresses this question 😄

Reply:
I can't keep up!